magento/adobe-stock-integration
# License asset that was saved as preview

### Description (*)
This PR introduces the possibility to license asset that was saved as preview. Once the asset is licensed, the preview image is being replaced with a full-sized image. 
Additionally, the functionality that hides the "License/License and Save" button for the licensed images, so we have only the "Locate" button. 

### Fixed Issues (if relevant)

1. #505: License asset that was saved as a preview 
2. #506: Locate licensed image

### Manual testing scenarios (*)
1. Login to admin panel
2. Open Magento Media Gallery (i.e. go to Cms -> Pages, edit the page and insert image)
3. Click "Search Adobe Stock" button to open images grid
4. Click on any image to expand image preview
5. Click on "Save Preview" button and save the preview
6. Reopen the Adobe Stock images grid and see the image preview section for just saved image
7. Click on "License" button
8. The preview image is replaced with the full-sized image
